### PR TITLE
Add binary messages

### PR DESCRIPTION
**Pull request by martinthomson**
I'm pleased to see that adding the file is all that is necessary.

I didn't retain the history of the file; this is a simple copy.  It's a bit of work to figure out
how to do that.

### CODE REVIEW

File: draft-ietf-httpbis-binary-message.md [hunk 1]
```diff
@@ -0,0 +1,634 @@
+---
+title: "Binary Representation of HTTP Messages"
+abbrev: Binary HTTP Messages
+docname: draft-ietf-httpbis-binary-message-latest
+category: std
+ipr: trust200902
+area: ART
+workgroup: HTTPBIS
```
Comment (mnot) [inline draft-ietf-httpbis-binary-message.md:8]: Just 'HTTP', not the short name

File: draft-ietf-httpbis-binary-message.md [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+  QUIC: RFC9000
+
+informative:
+
+
+--- abstract
+
+This document defines a binary format for representing HTTP messages.
+
+
```
Comment (mnot) [inline draft-ietf-httpbis-binary-message.md:38]: Add the Note to Readers?

File: draft-ietf-httpbis-binary-message.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-httpbis-binary-message.md:8]: I thought that we had to use the short name here.  I guess that this is display, not metadata.

File: draft-ietf-httpbis-binary-message.md [hunk 3]
```diff
@@ -0,0 +1,641 @@
...
+area: ART
+workgroup: HTTP
+venue:
+  group: HTTP
+  type: Working Group
+  home: https://httpwg.org/
+  mail: ietf-http-wg@w3.org
+  arch: https://lists.w3.org/Archives/Public/ietf-http-wg/
+  repo: https://github.com/httpwg/http-extensions/labels/binary-messages
+
```
Comment (mnot) [inline draft-ietf-httpbis-binary-message.md:16]: Needs `github-issue-label` too. (we might want to put that under  `venue` eventually, but that can be done separately.

**Pull request merged**